unicore_cli/train.py

```diff
@@ -49,6 +49,7 @@ def main(args) -> None:
     ), "Must specify batch size either with --batch-size"
     metrics.reset()
 
+    args.seed += args.dp_rank
```

Comment:
is this change needed?

Reply:
When using a hybrid distributed parallel strategy, such as DP-BP, the parameters and data in the same BP group need to be the same, so the seeds need to be the same.